### PR TITLE
Facade interfaces, DTOs, update entity classes

### PR DESCRIPTION
**Pull request by AdamKrajcik**
Do not merge!

### CODE REVIEW

File: api/src/main/java/cz/muni/fi/pa165/sportsclub/facade/TeamFacade.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    void updateTeam(TeamDto team);
+
+    void deleteTeam(TeamDto team);
+
+    void addPlayer(PlayerDto player, int jerseyNumber);
+
+    void removePlayer(PlayerDto player);
+
+    void chengeJerseyNumber(PlayerDto player);
```
Comment (HonzaCech) [inline api/src/main/java/cz/muni/fi/pa165/sportsclub/facade/TeamFacade.java]: typo - change
What number to change to - I guess there should be additional parameter.

File: api/src/main/java/cz/muni/fi/pa165/sportsclub/facade/TeamFacade.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    void getTeam(Long id);
```
Comment (HonzaCech) [inline api/src/main/java/cz/muni/fi/pa165/sportsclub/facade/TeamFacade.java]: In my opinion, get team by bussines key should be possible too

File: api/src/main/java/cz/muni/fi/pa165/sportsclub/facade/PlayerFacade.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ */
+public interface PlayerFacade {
+
+    void createPlayer(PlayerDto player);
+
+    void updatePlayer(PlayerDto player);
+
+    void deletePlayer(PlayerDto player);
+
+    void getPlayer(Long id);
```
Comment (HonzaCech) [inline api/src/main/java/cz/muni/fi/pa165/sportsclub/facade/PlayerFacade.java]: In my opinion, get player by bussines key (email, if I remember correctly) should be possible too (same for coach)

**Comment by AdamKrajcik** [comment]
I will look at the example project and try to repair the everything today. But also it's needed to look at reviews and repair tests for DAOs.

**Comment by skrovina** [comment]
@AdamKrajcik  Pull requests should be as small as possible, so they can be quickly merged. If the changes can be done in separate PR without much hassle, let's merge this one and create another PR.

**Comment by AdamKrajcik** [comment]
The repairs for DAOs should/will be done in a separate branch and pull request, but the repairs of facades should be done here.

**Comment by AdamKrajcik** [comment]
And someone look at the example project and try to find some differences.

**Pull request merged**